tf2onnx/tfonnx.py

```diff
@@ -185,18 +187,105 @@ def direct_op(ctx, node, name, args):
 def identity_op(ctx, node, name, args):
     """Identity."""
     if node.inputs[0].is_const():
-        # if identity has a const as input, remove it
+        # if identity has a const as input, remove it if it is not an model output
         input_name = node.input[0]
         output_name = node.output[0]
         for n in ctx.get_nodes():
             for i, parent_name in enumerate(n.input):
                 if parent_name == output_name:
                     n.input[i] = input_name
-        return None
+        if not ctx.is_model_output(output_name):
+            return None
+
     ctx.copy_shape(node.input[0], node.output[0])
     return node
 
 
+def range_op(ctx, node, name, args):
+    """Range."""
```

Comment:
list Range signature like this? 

    # T output = Select(bool condition, T x, T y)
    # V v_final_and_scan_outputs = Loop(int64 M, B cond, V v_initial)